src/index.js

```diff
@@ -194,7 +193,12 @@ export default class ToggleBlock {
     const { status } = this.data;
     const value = (children.length > 0 || status === 'closed');
 
-    lastChild.hidden = value;
+    if (value) {
+      lastChild.classList.add('hidden');
+    } else {
+      lastChild.classList.remove('hidden');
+    }
+
```

Comment:
@juliansg022 you can use the `toggleClass` method.

Reply:
It's a great method 😃